modules/transport.js

```diff
@@ -208,8 +206,8 @@ __private.receiveTransactions = function (query, peer, extraLogMessage, cb) {
 };
 
 /**
- * Normalizes transaction and remove peer if it fails.
- * Calls balancesSequence.add to receive transaction and 
+ * Normalizes transaction and bans peer if it fails.
```

Comment:
Should be: `Normalizes transaction and remove peer if it fails.` Banning mechanism has been removed.